README.md

```diff
@@ -1,23 +1,26 @@
+It can be used for synchronization of product (incl. price and inventory information), category and content data, and includes native storefront cartridges to power Search & Discovery Experiences on both [SFRA (Storefront Reference Architecture)](https://developer.salesforce.com/docs/commerce/sfra/guide/sfra-overview.html) and SiteGenesis. It also is compatible with Headless/Composable storefront integrations, incl. via the [PWA Kit](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/getting-started.html), accelerated by Algolia’s developer-friendly [InstantSearch](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/), [Autocomplete](https://www.algolia.com/doc/ui-libraries/autocomplete/introduction/what-is-autocomplete) and [Recommend](https://www.algolia.com/doc/ui-libraries/recommend/introduction/getting-started/?client=React) UI libraries. The integration can be used in conjunction with Algolia’s rich suite of Search & Discovery products such as its user-friendly [Merchandising Studio](https://www.algolia.com/doc/guides/managing-results/rules/merchandising-and-promoting/) and [Search Analytics](https://www.algolia.com/doc/guides/search-analytics/overview/), powerful [AI Search](https://www.algolia.com/doc/guides/getting-started/neuralsearch), (Personalization)[https://www.algolia.com/doc/guides/getting-started/neuralsearch/] and [Recommend](https://www.algolia.com/doc/guides/getting-started/neuralsearch/).
 
+To learn more about the cartridge and how to get started, check out our documentation for more information.
 
-### References
+## References
 - [Algolia Docs](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/getting-started/introduction/)
+- [AppExchange Listing](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000IkzoAUAR&tab=e)
 - [Academy: Algolia's Salesforce B2C Commerce Integration](https://academy.algolia.com/training/718dcbf0-786f-11ec-a21b-02d47b69d3fd/overview)
+- [General information](https://www.algolia.com/search-solutions/salesforce-commerce-cloud/)
 
----
+## Need Help?
 
-### Need Help?
-Here are some helpful documentation to help with your issue:
+Here is some helpful documentation that might answer some of your questions:
 
-- [Algolia Support Articles](https://support.algolia.com/hc/en-us)
-- [General FAQs](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/troubleshooting/faq/)
-- [Monitoring](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/troubleshooting/monitoring/)
-- [Product Indexing](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/indexing/product-indexing/overview/)
-- [Content Indexing](https://www.algolia.com/doc/integration/salesforce-commerce-cloud-b2c/indexing/content-indexing/fundamentals/)
+- FAQs
```

Comment:
Does this still contain the corresponding links for each of these?

Reply:
I'm ok to remove the section entirely, as the documentation has a proper sidebar with all the sections.